<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QWidget" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>140</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Paaikkuna</string>
  </property>
  <widget class="QSpinBox" name="spinBox_kmh">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>31</height>
    </rect>
   </property>
   <property name="suffix">
    <string> km/h</string>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber_mailiah">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_kmh">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_mailiah">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mailia/h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_nollaa">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Nollaa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_lopeta">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Lopeta</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_lopeta</sender>
   <signal>clicked()</signal>
   <receiver>Paaikkuna</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>261</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_kmh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_kmh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_kmh</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_kmh</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
